packages/app/cypress/e2e/integration/settings.spec.ts

```diff
@@ -8,95 +8,91 @@ describe('App: Settings', { viewportWidth: 600 }, () => {
+      // navigate away and come back
+      // preferred editor selected from dropdown should have been persisted
+      cy.visitApp()
+      cy.get('[href="#/settings"]').click()
+      cy.wait(100)
+      cy.get('[data-cy="Device Settings"]').click()
 
-    cy.contains('Choose your editor...').click()
-    cy.contains('Well known editor').click()
-    cy.wait('@SetPreferred').its('request.body.variables.value').should('include', '/usr/bin/well-known')
+      cy.get('[data-cy="use-well-known-editor"]').should('be.checked')
+      cy.get('[data-cy="use-custom-editor"]').should('not.be.checked')
+    })
 
-    // navigate away and come back
-    // preferred editor selected from dropdown should have been persisted
-    cy.visitApp()
-    cy.get('[href="#/settings"]').click()
-    cy.wait(100)
-    cy.get('[data-cy="Device Settings"]').click()
+    it('allows custom editor', () => {
+      cy.intercept('POST', 'mutation-ExternalEditorSettings_SetPreferredEditorBinary').as('SetPreferred')
 
-    cy.get('[data-cy="use-well-known-editor"]').should('be.checked')
-    cy.get('[data-cy="use-custom-editor"]').should('not.be.checked')
+      // doing invoke instead of `type` since `type` enters keys on-by-one, triggering a mutation
+      // for each keystroke, making it hard to intercept **only** the final request, which I want to
+      // assert contains `/usr/local/bin/vim'
+      cy.findByPlaceholderText('Custom path...').clear().invoke('val', '/usr/local/bin/vim').trigger('input').trigger('change')
+      cy.wait('@SetPreferred').its('request.body.variables.value').should('include', '/usr/local/bin/vim')
 
-    // change to custom editor using input
-    cy.findByPlaceholderText('Custom path...').clear().invoke('val', '/usr/local/bin/vim').trigger('input').trigger('change')
+      // navigate away and come back
+      // preferred editor entered from input should have been persisted
+      cy.get('[href="#/settings"]').click()
+      cy.wait(100)
+      cy.get('[data-cy="Device Settings"]').click()
 
-    // navigate away and come back
-    // preferred editor entered from input should have been persisted
-    cy.get('[href="#/settings"]').click()
-    cy.wait(100)
-    cy.get('[data-cy="Device Settings"]').click()
+      cy.get('[data-cy="use-well-known-editor"]').should('not.be.checked')
+      cy.get('[data-cy="use-custom-editor"]').should('be.checked')
+      cy.get('[data-cy="custom-editor"]').should('have.value', '/usr/local/bin/vim')
+    })
+
+    it('lists file browser as available editor', () => {
+      cy.intercept('POST', 'mutation-ExternalEditorSettings_SetPreferredEditorBinary').as('SetPreferred')
 
-    cy.get('[data-cy="use-well-known-editor"]').should('not.be.checked')
-    cy.get('[data-cy="use-custom-editor"]').should('be.checked')
+      cy.contains('Choose your editor...').click()
+      cy.get('[data-testid="computer"]').click()
```

Comment:
I started a discussion in Slack regarding the test attributes in use today: `data-cy`, `data-testid`, and `data-test`. There is a desire to standardize on `data-cy` given that is what is in our wheelhouse and what we recommend to our users.

The biggest hurdle to that is our usage of testing-library's Cypress command extensions, but we could [override](https://github.com/testing-library/cypress-testing-library#config-testidattribute) the attribute they're looking for to `data-cy` and continue to use the various ByTestId lookups.